swarms/utils/print_class_parameters.py

```
@@ -0,0 +1,36 @@
+
+    Args:
+        cls: The class object to introspect.
+        api_format: If True, return a dictionary mapping parameter names to the
+            string representation of their annotations. When False, the mapping
+            is also printed.
+
+    Raises:
+        Exception: If ``cls`` is not a class or has no ``__init__`` signature.
+    """
+    if not inspect.isclass(cls) or cls.__module__ == "builtins":
+        raise Exception("Input must be a user-defined class")
```

Pylint (3.3.7)

[warning] Raising too general exception: Exception

(W0719)